providers/gce/gce.go

```diff
@@ -94,6 +94,12 @@ var _ cloudprovider.Zones = (*Cloud)(nil)
 var _ cloudprovider.PVLabeler = (*Cloud)(nil)
 var _ cloudprovider.Clusters = (*Cloud)(nil)
 
+type StackType string
+
+const NetworkStackDualStack StackType = "IPV4_IPV6"
```

Comment:
Curious - does the order matter here? e.g. `IPV6_IPV4` (I vaguely remembered dualstack distinguishes that.)

Reply:
It does matter. IPV4_IPV6 is different from IPV6_IPV4. The first value indicates the default stack type.